cdap-docs/reference-manual/source/http-restful-api/http-restful-api-v3/logging.rst

```diff
@@ -63,15 +67,24 @@ API. To do that, send an HTTP GET request::
 
    * - Parameter
      - Description
-   * - ``<namespace>``
-     - Namespace ID
-   * - ``<component-id>``
-     - ``appfabric``
    * - ``<service-id>``
-     - [to be completed]
+     - One of ``appfabric``, ``dataset.executor``, ``explore.service``, ``metrics``, ``streams``
```

Comment:
These are the ones with "logs: OK" in the list that the list api returns. If there are ones that should have logs that don't, that would be a bug, but I have no way of knowing which ones should and shouldn't be available. Is there a way to tell what **should** be available?